app/src/main/java/chat/rocket/android/authentication/loginoptions/presentation/LoginOptionsPresenter.kt

```diff
@@ -72,20 +72,23 @@ class LoginOptionsPresenter @Inject constructor(
         credentialToken = oauthToken
         credentialSecret = oauthSecret
         loginMethod = AuthenticationEvent.AuthenticationWithOauth
+        setupConnectionInfo(currentServer)
```

Comment:
@filipedelimabrito I've already fixed this... You are double calling `setupConnectionInfo(currentServer)` now

Reply:
https://github.com/RocketChat/Rocket.Chat.Android/commit/09c179f1248786f6dbd2b4c715c949e7243b3e84